src/workerd/io/worker-interface.capnp

```diff
@@ -696,3 +697,19 @@ interface WorkerdBootstrap {
   # TODO(someday): Pass cfBlobJson? Currently doesn't matter since the cf blob is only present for
   #   HTTP requests which can be delivered over regular HTTP instead of capnp.
 }
+
+interface WorkerdDebugPort {
+  # Bootstrap interface exposed on the debug RPC port, if one is configured. This exposes access
+  # to all services in the process, with the ability for the client to specify arbitrary props, so
+  # this interface should be considered privileged, and should probably only be used for testing
+  # purposes.
+  #
+  # This interface is subject to change. It is intended for use by miniflare.
+
+  getEntrypoint @0 (service :Text, entrypoint :Text, props :Frankenvalue)
+              -> (entrypoint :WorkerdBootstrap);
+  # Get direct access to a stateless entrypoint.
+
+  getActor @1 (service :Text, entrypoint :Text, actorId :Data) -> (actor :WorkerdBootstrap);
```

Comment:
I wonder if we should make `actorId` be `Text` here -- and expect a hex string in the case of durable objects. Might be easier for callers to deal with? Note you can use `kj::decodeHex()` from `kj/encoding.h`.